plugin/parser/edit/pg/deparse.go

```diff
@@ -143,55 +143,61 @@ func transformAlterTableContext(ctx *DeparseContext, alterTableContext *api.Alte
 			ColumnList: []*ast.ColumnDef{columnDef},
 		})
 		if addColumnContext.Comment != "" {
-			commemtStmt := fmt.Sprintf("COMMENT ON COLUMN %s.%s IS '%s';", table.Name, columnDef.ColumnName, addColumnContext.Comment)
+			commemtStmt := fmt.Sprintf(`COMMENT ON COLUMN "%s"."%s"."%s" IS '%s';`, table.Schema, table.Name, columnDef.ColumnName, addColumnContext.Comment)
 			ctx.StmtList = append(ctx.StmtList, commemtStmt)
 		}
 	}
 
-	for _, changeColumnContext := range alterTableContext.ChangeColumnList {
-		if changeColumnContext.OldName != changeColumnContext.NewName {
+	for _, alterColumnContext := range alterTableContext.AlterColumnList {
+		if alterColumnContext.OldName != alterColumnContext.NewName {
 			alterTableStmt.AlterItemList = append(alterTableStmt.AlterItemList, &ast.RenameColumnStmt{
```

Comment:
RenameColumnStmt should use a single AlterTableStmt, which means it's AlterItemList = {RenameColumnStmt}

FYI https://www.postgresql.org/docs/current/sql-altertable.html

Reply:
Updated